css/index.css

```diff
@@ -0,0 +1,66 @@
+body {
+   background-color: #FBF2C4; 
+   color: #008585;
+   margin: 40px, 100, 120px, 80px;
+   padding: 50px;
+}
+head {
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+h1 {
+    color:#C7522A;
+}
+h2 {
+    color: #C7522A;
+    font-size: 24px;
+    font-weight: 800;
+}
+#experience {
+    system: fixed;
+    border-block-style: groove;
+}
+#mySkills {
+    list-style: custom-ordered;
```

Comment:
`custom-ordered` is pretty esoteric. You have have copied this from an example? That's fine, but you should understand what each part does.